vsintegration/src/FSharp.Editor/CodeFixes/ImplementInterfaceCodeFixProvider.fs

```diff
@@ -32,15 +30,19 @@ type internal InterfaceState =
         Tokens: Tokenizer.SavedTokenInfo[]
     }
 
-[<ExportCodeFixProvider(FSharpConstants.FSharpLanguageName, Name = "ImplementInterface"); Shared>]
+// state machine not statically compilable
+// TODO: rewrite token arithmetics properly here
```

Comment:
Is this TODO for later?

Reply:
Yeah. For those who like doing token arithmetics. There should be at least a few folks for that on this planet.

Reply:
Ok, I usually use TODO for stuff I mean to resolve within given PR in a future commit, hence the question